mender/templates/deployments/configmap-prerelease.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and .Values.deployments.enabled .Values.deployments.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mender.fullname" . }}-deployments
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "mender.labels" . | nindent 4 }}
+    app.kubernetes.io/name: {{ include "mender.fullname" . }}-deployments-prerelease
+    app.kubernetes.io/component: deployments
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-weight: "-10"
```

Comment:
Wouldn't it be easier to just add the `helm.sh/hook` and `helm.sh/hook-weight` to the existing configmap to ensure it's created before the migration job?